infoschema/metric_schema.go

```diff
@@ -925,4 +925,1055 @@ var MetricTableMap = map[string]MetricTableDef{
+	},
+	"tikv_leader_missing": {
+		PromQL:  `sum(tikv_raftstore_leader_missing{$LABEL_CONDITIONS}) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The count of missing leaders per TiKV instance",
+	},
+	"tikv_active_written_leaders": {
+		PromQL:  `sum(rate(tikv_region_written_keys_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The number of leaders being written on each TiKV instance",
+	},
+	"tikv_approximate_region_size": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_raftstore_region_size_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,instance))`,
+		Labels:   []string{"instance"},
+		Quantile: 0.99,
+		Comment:  "The approximate Region size",
+	},
+	"tikv_approximate_avg_region_size": {
+		PromQL:  `sum(rate(tikv_raftstore_region_size_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) / sum(rate(tikv_raftstore_region_size_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) `,
+		Labels:  []string{"instance"},
+		Comment: "The approximate avg Region size",
+	},
+	"tikv_approximate_region_size_histogram": {
+		PromQL: `sum(rate(tikv_raftstore_region_size_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_region_average_written_bytes": {
+		PromQL:  `sum(rate(tikv_region_written_bytes_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance) / sum(rate(tikv_region_written_bytes_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The average rate of writing bytes to Regions per TiKV instance",
+	},
+	"tikv_region_written_bytes": {
+		PromQL: `sum(rate(tikv_region_written_bytes_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_region_average_written_keys": {
+		PromQL:  `sum(rate(tikv_region_written_keys_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance) / sum(rate(tikv_region_written_keys_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "The average rate of written keys to Regions per TiKV instance",
+	},
+	"tikv_region_written_keys": {
+		PromQL: `sum(rate(tikv_region_written_keys_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION]))`,
+		Labels: []string{"instance"},
+	},
+	"tikv_request_batch_avg": {
+		PromQL:  `sum(rate(tikv_server_request_batch_ratio_sum{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance) / sum(rate(tikv_server_request_batch_ratio_count{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (type,instance)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "The ratio of request batch output to input per TiKV instance",
+	},
+	"tikv_request_batch_ratio": {
+		PromQL:   `histogram_quantile($QUANTILE, sum(rate(tikv_server_request_batch_ratio_bucket{$LABEL_CONDITIONS}[$RANGE_DURATION])) by (le,type,instance))`,
+		Labels:   []string{"instance", "type"},
+		Quantile: 0.99,
+		Comment:  "The ratio of request batch output to input per TiKV instance",
```

Comment:
The comment is the same as above, and key name 'tikv_request_batch_ratio' doesn't indicate any information about quantile.

Reply:
done.